rholang/src/main/scala/coop/rchain/rholang/ResourceTools.scala

```diff
@@ -25,4 +30,29 @@ object ResourceTools {
           case _ => new Directory(new File(path.toString)).deleteRecursively()
         })
     )
+
+  def mkRhoISpace[F[_]: Sync, A](
+      prefix: String = "",
+      branch: String = "test",
+      mapSize: Long = 1024L * 1024L * 4
+  ): Resource[F, RhoISpace[F]] = {
+    import coop.rchain.rholang.interpreter.storage.implicits._
+
+    def mkRspace(dbDir: Path): F[RhoISpace[F]] = {
+      val context: RhoContext = Context.create(dbDir, mapSize)
```

Comment:
I think that `RhoContext` has a `close` method which should be called in the `release` part of the `Resource`.

Reply:
Fixed